src/adagrams.js

```diff
@@ -1,6 +1,110 @@
+import { ENGINE_METHOD_DIGESTS } from "constants";
+
+const hashIntoObject = arrOfLetters => {
+  let object = {};
+  arrOfLetters.forEach(letter => {
+    object[letter] ? object[letter] += 1 : object[letter] = 1;
+  });
+  return object;
+}
+
 const Adagrams = {
   drawLetters() {
     // Implement this method for wave 1
+    let letterPool = 'AAAAAAAAABBCCDDDDEEEEEEEEEEEEFFGGGHHIIIIIIIIIJKLLLLMMNNNNNNOOOOOOOOPPQRRRRRRSSSSTTTTTTUUUUVVWWXYYZ'.split('');
+    let lettersInHand = [];
```

Comment:
This big string of letters gets the job done, but it's not easy to tell at a glance how many there are of each letter. What if you instead kept a hash of letter frequencies, and used that to build a pool of letters in this function?